pilot/pkg/model/authentication.go

```diff
@@ -42,19 +42,14 @@ func GetConsolidateAuthenticationPolicy(mesh *meshconfig.MeshConfig, store Istio
 		}
 	}
 
-	legacyPolicy := consolidateAuthPolicy(mesh, port.AuthenticationPolicy)
+	legacyPolicy := meshAuthPolicy(mesh)
```

Comment:
You can remove meshAuthPolicy function, and use `mesh.AuthPolicy` directly.

Reply:
Refactored to directly use legacyAuthenticationPolicyToPolicy(), let me know if you're thinking something else.